src/prefect/deployments/runner.py

```diff
@@ -116,6 +122,12 @@ class RunnerDeployment(BaseModel):
             " this deployment."
         ),
     )
+    storage: Optional[RunnerStorage] = Field(
+        default=None,
+        description=(
+            "The storage object used to retrieve flow code for this deployment."
```

Comment:
I'd like to discuss expanding our view of storage a little - will be easier to discuss offline

Reply:
For posterity, `RunnerStorage.sync` was changed to `RunnerStorage.pull_code`. This will give us some space to add additional functionality to `RunnerStorage` in the future.